UndertaleModTool/Editors/UndertaleTexturePageItemEditor.xaml.cs

```diff
@@ -40,6 +40,9 @@ public UndertaleTexturePageItemEditor()
 
         private void UpdateImages(UndertaleTexturePageItem item)
         {
+            if (item.TexturePage?.TextureData?.Image is null)
```

Comment:
Same as above, copying for clarity:
If the `Image` property itself is set to null, we may want to actually clear the image sources in the UI, so they disappear.

Reply:
As above: I think I added what you were after. Is that the way you wanted it done?